knpu/episode1/tests-clear-data.md

```diff
@@ -80,7 +80,7 @@ Because we have the container, we have the EntityManager. And that also means
 we have an easy way to clear data. Create a new private function called `purgeDatabase()`.
 Because we have the Doctrine [DataFixtures](https://github.com/doctrine/data-fixtures)
 library installed, we can use a great class called `ORMPurger`. Pass it the
-EntityManager - so `$this->getService('doctrine')->getManager()`. To clear
+EntityManager - so `$this->getService('doctrine.orm.default_entity_manager')`. To clear
```

Comment:
you're right - but the best fix is to change the code block (so that it matches the video/script). Unfortunately, due to the fancy way we build code-blocks, that's not possible yet for anyone to do but me - the downside of new tech :). I'll update this soon - I'm going to leave this PR open until then to make sure it doesn't get lost.

Thanks for catching this!